syntax = "proto3";
package minecpp.proto.event.serverbound;

import "google/protobuf/any.proto";
import "minecpp/proto/player/Id.proto";

message Event {
  google.protobuf.Any payload = 1;
  player.PlayerId player_id = 2;
}

message RawMessage {
  bytes data = 1;
}

message AcceptPlayer {
  string name = 1;
  uint32 challenge_id = 2;
  uint32 front_id = 3;
}

message RemovePlayer {
}

message PreInitialChunks {
}

message PostInitialChunks  {
}
